engines/group/spec/controllers/groups_controller_spec.rb

```diff
@@ -1,8 +1,8 @@
 require 'rails_helper'
-require_relative '../../../support/shared_examples/controllers.rb'
-require_relative '../../../support/authentication.rb'
+require 'support/shared_examples/controllers.rb'
+require 'support/authentication.rb'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.